src/terra-dev-site/IssueForm/Helper.jsx

```diff
@@ -0,0 +1,117 @@
+import PropTypes from 'prop-types';
+import Packages from './Packages.json';
+
+// Prop types for all components.
+const propTypes = {
+  /**
+   * List of packages from terra clinical, core, and framework.
+   */
+  packageList: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * Functions that set state with the provided field values.
+   */
+  setContext: PropTypes.func,
+  setDescription: PropTypes.func,
+  setEnvironment: PropTypes.func,
+  setExpected: PropTypes.func,
+  setIssue: PropTypes.func,
+  setMentions: PropTypes.func,
+  setPackage: PropTypes.func,
+  setSolution: PropTypes.func,
+  setSteps: PropTypes.func,
+  setTitle: PropTypes.func,
+  /**
+   * Sets the value of the issue type select box to the selected issue type.
+   */
+  value: PropTypes.string,
+};
+
+// Templates used for preview, errors, and forms.
+const bugTemplate = props => `# Bug Report
+
+## Description
+${props.description}
+
+## Steps to Reproduce
+${props.steps}
+
+## Expected Behavior
+${props.expected}
+
+## Environment
+${props.environment}
+
+${props.context ? `## Additional Context / Screenshots
+${props.context}` : ''}
+
+${props.solution ? `## Possible Solution
+${props.solution}` : ''}
+
+${props.mentions ? `## Mentions
+${props.mentions}` : ''}`;
+
+const disclaimerTemplate = `When logging issues to Terra UI, it is helpful to have steps to reproduce the issue using a reduced test case, or examples from <a href="https://engineering.cerner.com/terra-ui" target="_blank">https://engineering.cerner.com/terra-ui</a>.
+    If there are videos or screenshots that are available, those are helpful as well.
+
+**Note to Cerner associates:** If you have proprietary information, don't post it directly.  Instead post a link to a site that can be protected by the Cerner IDP (e.g. sharepoint).`;
+
+const environmentTemplate = `* Component Version:
+* Browser Name and Version:
+* Node/npm Version [e.g. Node 8/npm 5]:
+* Webpack Version:
+* Operating System and version (desktop or mobile): `;
+
+const featureTemplate = props => `# Feature Request
```

Comment:
I think it would be cleaner if this was its own file

Reply:
Changed: [5eff56a](https://github.com/cerner/terra-ui/pull/151/commits/5eff56a5761ea4b8acbfe97a933e639186f91e48)